### PR TITLE
added blog post for yash sharma

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 15e963b158196fa5eb3adec45ac33c5da3c71405 at: https://67267b8ee00adf0a07c158fa--layer5.netlify.app

**Review by Yashsharma1911 — COMMENTED** [review]
Thankyou @hargunkaur286 I think we still missing the feedback given by Lee on email

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Meshery is growing very fast, it is the 9th fastest growing project under
+    CNCF. There are many opportunities are open for you to contribute. Start
+    with joining community and introducing yourself and sharing your skill. This
+    will help community managers to put you in right direction based on your
+    skills. All areas are consistently in need of help, including best at this
+    time to contribute like Meshery's CLI client, Meshery Server and digital
+    marketing.
```
Comment (Yashsharma1911) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:186]:
```suggestion
    Meshery is growing very fast, it is the 9th fastest growing project under
    CNCF. There are many opportunities are open for you to contribute. Start
    with joining community and introducing yourself and sharing your skill. This
    will help community managers to put you in right direction based on your
    skills. All areas are consistently in need of help, including best at this
    time to contribute like Meshery's CLI client, Meshery Server and digital
    marketing.
    
    Also out of 100+ repos that the Layer5 community stewards, two are currently closed: Kanvas and Cloud. Both of these projects were created and are actively being developed by open source contributors, though. There are ~15 open source contributors who working on it now, who meet daily. New contributors in the community routinely express interest and are invited to participate, and are extremely well supported as they do.

In other words, the source is currently closed (subject to change), but any community member here that shows interest, demonstrates competence and consistency of participation is readily invited to join in.
Please signal if this is you, and community managers will get you a tour of the code, introductions to other core team members, so that you might better assess your interest.
```

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>What is your favorite feature in Meshery UI?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Meshery is an extensible platform and I like this feature of it, it gives
```
Comment (Yashsharma1911) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:146]: We can include meshery.io link here for "Meshery" word

**Comment by l5io** [comment]
🚀 Preview for commit 9918899d9631951c3884fa4a82b93c6faf2ec991 at: https://67272d87d58019c14e406b61--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a98016044f6a5fa645e50eece7e0e15ce2d7e013 at: https://67273dd8a751f5b7b048beca--layer5.netlify.app

**Review by Yashsharma1911 — APPROVED** [review]

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    figured out how to start contributing, and the community managers were super
+    helpful and always around to give a hand. You just don’t see that kind of
+    support in a lot of other open source projects.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>
+    <Link to="/projects">Layer5 projects</Link> has a number of active, open
```
Comment (vishalvivekm) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:108]:
```suggestion
    <Link to="/projects">Layer5 projects</Link> have a number of active, open
```

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+<div class="interviewer">
+  <span>Hargun:</span>
+  <p>What’s the coolest Meshery demo you have done/seen?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Haha good one, this year I got a chance to represent Meshery at KubeCon
+    which was really an amazing experience for me. I gave a demo of Kavas and
```
Comment (vishalvivekm) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:132]:
```suggestion
    which was really an amazing experience for me. I gave a demo of Kanvas and
```

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+  <span>Hargun:</span>
+  <p>What is your favorite feature in Meshery UI?</p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    <Link to="https://meshery.io/">Meshery</Link> is an extensible platform and I like this feature of it, it gives Meshery a whole new powerful capability to interact with various
+    technologies in the ecosystem and use it as an extension. My favourite one
+    is Kavas, though.
```
Comment (vishalvivekm) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:148]:
```suggestion
    is Kanvas, though.
```

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+    Where do you see opportunities for contributors to get involved within
+    Meshery and Layer5 community?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Yash:</span>
+  <p>
+    Meshery is growing very fast, it is the 9th fastest growing project under
+    CNCF. There are many opportunities are open for you to contribute. Start
```
Comment (vishalvivekm) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:180]:
```suggestion
    CNCF. There are many opportunities open for you to contribute. Start
```

File: src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+    with joining community and introducing yourself and sharing your skill. This
```
Comment (vishalvivekm) [inline src/collections/blog/2024/11-01-meet-the-maintainer-yash-sharma/post.mdx:181]:
```suggestion
    with joining community, introducing yourself and sharing your skill. This
```

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
typos need fixed.

**Comment by vishalvivekm** [comment]
@hargunkaur286 Let's discuss this on websites call on Monday at 6:30 PM IST (7:00 AM CT). Add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk), if you would :)

**Comment by sudhanshutech** [comment]
@hargunkaur286 are vivek's suggestions done?

**Comment by hargunkaur286** [comment]
> @hargunkaur286 are vivek's suggestions done?

@sudhanshutech yes
